src/views/vmd-rdv.view.ts

```diff
@@ -36,6 +38,59 @@ export class VmdRdvView extends LitElement {
     @property({type: Array, attribute: false}) lieuxParDepartement: LieuxParDepartement | undefined = undefined;
     @property({type: Boolean, attribute: false}) searchInProgress: boolean = false;
 
+    @property({type: String, attribute: false}) critèreDeTri: 'date' | 'distance' = 'date'
+    @property({type: Boolean, attribute: false}) geolocalisationBloquée = false
+    @property({type: Boolean, attribute: false}) geolocalisationIndisponible = false
+    @property({type: Boolean, attribute: false}) afficherMessageGeoloc = false
+    private userLocation: Coordinates | undefined
+
+    get lieuxDisponiblesTriés () {
```

Comment:
Plutôt qu'une property, je ferais ce traitement dans le `refreshLieux()` afin d'éviter d'avoir à faire ce calcul 6 fois lors de chaque re- `render()`

Reply:
en fait on a pas forcément la localisation au moment du refreshLieux()

Reply:
Je ferais simplement un test dans le `refreshLieux` pour trier : 
- si `this.userLocation !== undefined && this.critèreDeTri === 'distance'` => tri par distance 
- sinon : tri par date de premier rdv dispo

Reply:
Après, tu peux aussi externaliser la fonctionnalité de sorting pour :
- L'appeler systématiquement lors d'un `refreshLieux()`
- N'appeler que elle sur un changement du `this.critèreDeTri`

Reply:
j'ai check et en fait lit-html à l'air d'être suffisamment futé pour ne pas faire d'appel non nécessaires :)